src/sidebar/index.js

```diff
@@ -131,7 +131,7 @@ import { ThreadsService } from './services/threads';
 import { ToastMessengerService } from './services/toast-messenger';
 
 // Redux store.
-import store from './store';
+import { createSidebarStore } from './store';
```

Comment:
This is also used by the `Notebook`, right?

Reply:
Yes. Currently the sidebar and the notebook share the same store. In future we might change this so that the sidebar and notebook app uses stores with different sets of modules.